frontend/components/organisms/QuestionList/index.tsx

```diff
@@ -53,12 +74,11 @@ const QuestionList = ({
             </div>
             <div className="flex w-[85%] flex-col gap-4">
                 <div className="flex w-full justify-between">
-                    <Link
-                        href={`/questions/${slug}`}
-                        className="text-lg text-blue-600 hover:text-blue-400"
-                    >
-                        {title}
-                    </Link>
+                    {page_slug == 'teams'
+                        ? renderTeamQuestionDetailHeader()
+                        : page_slug == 'questions'
+                        ? renderQuestionDetailHeader()
+                        : renderQuestionDetailHeader()}
```

Comment:
Can you confirm if both functions are the same? If yes, please refactor to eliminate redundancy.

Maybe try
`page_slug == 'questions' && renderQuestionDetailHeader()`